docs/index.rst

```diff
@@ -19,52 +19,48 @@ user-defined types among other features.
 With Solidity you can create contracts for uses such as voting, crowdfunding, blind auctions,
 and multi-signature wallets.
 
-.. note::
-    The best way to try out Solidity right now is using
-    `Remix <https://remix.ethereum.org/>`_
-    (it can take a while to load, please be patient). Remix is a web browser
-    based IDE that allows you to write Solidity smart contracts, then deploy
-    and run the smart contracts.
+Language Documentation
+----------------------
+
+If you are new to the concept of smart contracts we recommend you start with :ref:`an example smart contract <simple-smart-contract>` written
+in Solidity, followed by :ref:`the basics of blockchains <blockchain-basics>`
+and details of the :ref:`Ethereum Virtual Machine <the-ethereum-virtual-machine>`.
+
+When you are ready for more detail, we recommend you read the :doc:`"Solidity in Depth" <solidity-in-depth>` and :doc:`"Solidity by Example" <solidity-by-example>` sections to learn the core concepts of the language.
+
+Smart contracts and blockchains have their own unique issues to watch out for,
+so before deploying your contract, make sure you read the
+:ref:`security_considerations` section.
+
+.. hint::
+  You can always try out code examples in your browser with the
+  `Remix IDE <https://remix.ethereum.org>`_. Remix is a web browser based IDE
+  that allows you to write Solidity smart contracts, then deploy and run the
+  smart contracts. It can take a while to load, so please be patient.
+
+If you have any questions, you can try searching for answers or asking on the
+`Ethereum Stackexchange <https://ethereum.stackexchange.com/>`_, or our `gitter channel <https://gitter.im/ethereum/solidity/>`_.
+
+Ideas for improving Solidity or this documentation are always welcome, read our :doc:`contributors guide <contributing>` for more details.
 
 .. warning::
-    Since software is written by humans, it can have bugs. Thus, also
-    smart contracts should be created following well-known best-practices in
-    software development. This includes code review, testing, audits and correctness proofs.
-    Also note that users are sometimes more confident in code than its authors.
-    Finally, blockchains have their own things to watch out for, so please take
-    a look at the section :ref:`security_considerations`.
+    Humans write software, and it can have bugs. You should follow established
```

Comment:
I think we should keep this at a rather prominent place. Also I pretty much liked the comparison between non-blockchain related code and blockchain-related code. Writing smart contract code and writing self-driving car code should have the same security standards.

Reply:
OK, it felt a little overbearing to me, but point taken. For me it was more about the flow. It seemed odd to tell people to pay attention to security considerations for a language they don't even know anything about yet. So try this change.

Reply:
Why do you assume they don't know anything about it? This is the landing page of the documentation, it does not mean that all people coming here are totally new to the concept.